src/Workspaces/Core/Portable/Diagnostics/CompilationWithAnalyzersPair.cs

```diff
@@ -44,8 +44,6 @@ public CompilationWithAnalyzersPair(CompilationWithAnalyzers? projectCompilation
 
     public CompilationWithAnalyzers? HostCompilationWithAnalyzers => _hostCompilationWithAnalyzers;
 
-    public bool ReportSuppressedDiagnostics => _projectCompilationWithAnalyzers?.AnalysisOptions.ReportSuppressedDiagnostics ?? _hostCompilationWithAnalyzers!.AnalysisOptions.ReportSuppressedDiagnostics;
```

Comment:
>ReportSuppressedDiagnostics [](http://example.com/codeflow?start=97&length=27)

Is this flag used anymore (other than in the contract check on line 27)?

Reply:
this is all defined at the compiler layer.  so can't remove :)

Reply:
Definitely understand not wanting to do so in this PR. Maybe a followup compiler change if it's obvious that removal is safe?

Reply:
sure!